packages/manager/modules/mfa-enrollment/src/mfa-enrollment.controller.js

```diff
@@ -0,0 +1,32 @@
+import get from 'lodash/get';
+import {
+  DEFAULT_ASSET,
+  DEFAULT_ASSET2X,
+  DEFAULT_ASSET3X,
+} from './mfa-enrollment.constants';
+
+export default class mfaEnrollmentCtrl {
+  /* @ngInject */
+  constructor(
+    $state,
+    CORE_MANAGER_URLS,
+    coreConfig,
+  ) {
+    this.$state = $state;
+    const mfaBaseUrl = get(CORE_MANAGER_URLS, `${coreConfig.getRegion()}.dedicated`);
```

Comment:
Then you have a peer dependency with [@ovh-ux/manage-core](https://www.npmjs.com/package/@ovh-ux/manager-core) package I guess.